tools/timeit.c

```diff
@@ -219,7 +407,9 @@ static int monitor_child_process(pid_t pid, double start_time) {
     fprintf(fp, "%-10s %.4f\n", "sys", sys_time);
     fclose(fp);
   }
-
+#ifdef _WIN32
+  CloseHandle(h_output_file);
```

Comment:
Shouldn't we have an `if (h_output_file != INVALID_HANDLE_VALUE)` around this?

Reply:
yes. I have fixed this.